commands/audit/sca/common.go

```diff
@@ -21,6 +21,11 @@ import (
 	xrayUtils "github.com/jfrog/jfrog-client-go/xray/services/utils"
 )
 
+const (
+	// Visual Studio inner directory to exclude when copying the project to a temporary directory during the dependency tree build process.
```

Comment:
```suggestion
```
When writing an explanation above a const we want to explain what the const is, and not what it is used for at definition, since for a developer that is not familiar with the context it will be hard to understand what you ment.
We want to explain about a "strange" behaviour or something that needs extra details at the stop where it occurs